client/src/partials/journal/journal.html

```diff
@@ -2,7 +2,21 @@
   <div class="bhima-title">
     <ol class="headercrumb">
       <li class="static">{{ "TREE.FINANCE" | translate }}</li>
-      <li class="title">{{ "POSTING_JOURNAL.TITLE" | translate }}</li>
+      <li class="title">
+        {{ "POSTING_JOURNAL.TITLE" | translate }}
+
+        <span class="badge badge-primary text-uppercase" ng-show="JournalCtrl.transactions.isEditing()">
+          <i class="fa fa-street-view"></i>
+          Edit Mode
+        </span>
+
+        <span class="badge badge-warning text-uppercase" ng-show="JournalCtrl.transactions._highlights.length > 0">
+          {{ JournalCtrl.transactions._highlights.length }}
+          <span ng-if="JournalCtrl.transactions._highlights.length == 1">Highlight</span>
```

Comment:
Are highlights a feature that will be implemented in the future or that were used during development/ debugging? 
I can't see any way that this state is ever reached in this PR.